base/applications/games/solitaire/solitaire.cpp

```diff
@@ -738,7 +738,7 @@ LRESULT CALLBACK WndProc (HWND hwnd, UINT iMsg, WPARAM wParam, LPARAM lParam)
                 return 0;
 
             case IDM_HELP_ABOUT:
-                MessageBox(hwnd, MsgAbout, szAppName, MB_OK|MB_ICONINFORMATION);
+                ShellAboutW(hwnd, szAppName, MsgAbout, LoadIconW(hInstance, MAKEINTRESOURCE(IDI_SOLITAIRE)));
```

Comment:
I don't care too much about "long" lines,
but 'W' are wrong, aren't they?

Reply:
My understanding is that 'W' versions of functions are preferred. When using the 'W' versions of functions, you explicitly specify that you want the Unicode version of the function which is what we will always want on NT-like OS's like ReactOS.

Reply:
Exactly. But you have to ensure that the passed string buffers (szAppName, etc.) are explicitly WCHAR or the like (PWSTR etc.)